custom_card.json

```diff
@@ -0,0 +1,10 @@
+{
+  "color": "red",
+  "actions": {
+    "skip": true,
+    "reverse": true,
+    "swap": true,
+    "change color": true
+  },
+  "draw penalty": 2
+}
```

Comment:
Hey Tise, looks great! But like from the last review, I think you should consider making an additional entry in the JSON that can give information about how many of this particular card to include in the deck. It would be really helpful if this was included to reduce the size of JSON files, especially since multiple of the same card could exist in a deck (for example, multiple wild cards, +4 cards, +2 cards, etc). That is the only change needed.

Reply:
okay. I'll add that